### ISSUE TITLE
`es.indices.put_alias` & `es.indices.get_alias` doesn't work

### ISSUE THREAD

**Issue by idrissbellil** [vrcmarcos/elasticmock#39]
Judging from an other issue, it's not maintained but I hope you still accept PRs. I will submit one ASAP.

**Comment by vrcmarcos** [comment]
We're still accepting PRs :)

**Comment by cuent** [comment]
now are aliases working ?

**Comment by vrcmarcos** [comment]
I don't believe that anyone has sent a PR adding this feature, but it would be awesome if you could do it 😀
